components/CandidateList/CandidateList.tsx

```diff
@@ -1,19 +1,36 @@
 ﻿import React, {useEffect, useState} from "react";
-import scss from './CandidateList.module.scss';
+import scss from "../../pageStyles/candidates.module.scss";
 import {Candidate, getCandidates, ListResponse} from "../../api/candidatesApiClient";
 import {CandidateLink} from "../CandidateLink/CandidateLink";
+import Router , {useRouter}  from 'next/router';
 
 interface CandidatesListProps {
     candidatesList: ListResponse<Candidate>;
 }
 
 export function CandidateList(props: CandidatesListProps): JSX.Element {
     const candidates=props.candidatesList.items;
+
+    function decrementPage(){
+        Router.push(`/candidates?page=${props.candidatesList.page - 1}&pageSize=10`);
+    }
+
+    function incrementPage(){
+        Router.push(`/candidates?page=${props.candidatesList.page + 1}&pageSize=10`);
```

Comment:
don't think you need the `pageSize` param :)

Reply:
using pageSize param now